### PR TITLE
Elbow Dislocations

### PR DESCRIPTION
**Pull request by andyglittle**
Don't have the image loaded yet.

### CODE REVIEW

**Comment by andyglittle** [comment]
Here is the image

image

File: cards/Elbow Dislocations [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+
+title: Elbow Injuries- Elbow Dislocations
```
Comment (jvoros) [inline cards/Elbow Dislocations]: Needs a space `Elbow Injuries - Elbow Dislocaitons`

File: cards/Elbow Dislocations [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - Mary Haas, MD
+created: 2016/12/12
+updates: null
+categories:
+    - Peer Reviewed
+    - Orthopedics
+
+---
+
+ #Elbow Injuries: Elbow Dislocation
```
Comment (jvoros) [inline cards/Elbow Dislocations]: Headers need a space: `# Elbow Injuries...`

**Comment by andyglittle** [comment]
Jeremy I'll get this done tonight

**Comment by andyglittle** [comment]
Jeremy,

    I've been trying to build folders for the past few nights when I;ve
gotten off work. I reset up my forks, and get to the load folder section
and am not sure what I should be dragging and dropping. Can we set up a
time to screen share?

Andy

**Comment by dsifford** [comment]
@andyglittle This branch still needs to have the card placed into its own folder.

Thanks 👍

**Pull request closed without merging**